pkg/execution/state/testharness/testharness.go

```diff
@@ -39,25 +33,10 @@ var (
 		},
 		Steps: []inngest.Step{
 			{
-				ID:   "step-a",
+				ID:   inngest.DefaultStepName,
 				Name: "first step",
 				URI:  "http://www.example.com/api/inngest",
 			},
-			{
-				ID:   "step-b",
-				Name: "second step",
-				URI:  "http://www.example.com/api/inngest",
-			},
-		},
-		Edges: []inngest.Edge{
-			{
```

Comment:
Are we losing test cases here or is this refactor equivalent?

Reply:
Not relevant any more.